ehcache-impl/src/main/java/org/ehcache/impl/events/EventDispatchTask.java

```diff
@@ -40,12 +40,10 @@
   @Override
   public void run() {
     for(EventListenerWrapper<K, V> listenerWrapper : listenerWrappers) {
-      if (listenerWrapper.isForEventType(cacheEvent.getType())) {
```

Comment:
Why this isForEventType check removed?

Reply:
listenerWrappers are already correctly preselected to only contain relevant entries from *syncListenersList.get(event.getType()) ?